Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -130,8 +130,8 @@ class BaseTypeTests : XCTestCase {
     }
 }
 
-let didOverflow = ArithmeticOverflow(true)
-let didNotOverflow = ArithmeticOverflow(false)
+let didOverflow = true
+let didNotOverflow = false
```

Comment:
Could you pull in the `didOverflow` and `didNotOverflow` values into the calling sites? I had them pulled out because of the verbosity of writing out `ArithmeticOverflow(true/false)` all over my tests, but with that requirement gone, I think it would be nicer to pull all of these variables in.